### ISSUE TITLE
[A11y] Copy modal and Delete popovers in Timeline pages

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6423]
### Description

From the accessibility checks performed on timeline related pages in https://github.com/avalonmediasystem/avalon/issues/6314, the following issue was identified for copy modal and delete popovers in timeline pages.

| Success criterion | WCAG 2.1 Level | Accessibility checker | Error | Relevant HTML element |
| --- | --- | --- | --- | --- |
| 4.1.2 Name, Role, Value | Level A | SiteImprove | Button missing a text alternative | Close button of the copy timeline modal |
| 4.1.2 Name, Role, Value | Level A | SiteImprove & WAVE| Form field missing a text alternative | `<input>` textbox for  timeline `Name` |
| ARIA-authoring practices | N/A | SiteImprove & axeDevtool | ARIA attribute does not exist | `aria-label` is misspelled for the close button of the copy modal |
| ARIA-authoring practices | N/A | SiteImprove | Text not included in an ARIA landmark | Text and buttons inside the delete popover |

### Done Looks Like
- [ ] All identified issues are fixed and SiteImprove and other accessibility tools do not complain about UI elements related to copy modal and delete popover

**Comment by joncameron** [comment]
"Text not included in an ARIA landmark" may be more difficult and require some research.
